CHANGELOG.md

```
@@ -18,6 +18,10 @@ and this project adheres to
 - ✨(backend) order pinned documents by last updated at #2028
 - 🛂(frontend) fix cannot manage member on small screen #2226
 
+### Changed
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)